WordPress/Sniffs/VIP/SuppressFiltersSniff.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Checks for suppress_filters=>false being supplied in get_posts(), wp_get_recent_posts() and get_children().
+ *
+ * @link    https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.14.0
+ */
+class WordPress_Sniffs_VIP_SuppressFiltersSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * Functions this sniff is looking for.
+	 *
+	 * @since 0.14.0
+	 *
+	 * @var array
+	 */
+	protected $target_functions = array(
+		'get_posts'           => true,
+		'wp_get_recent_posts' => true,
+		'get_children'        => true,
+	);
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @since 0.14.0
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 * @param array $group_name The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array $parameters Array with information about the parameters.
+	 *
+	 * @return void
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+
+		if ( false === $this->target_functions[ $matched_content ] ) {
+			return;
+		}
+
+		// Flag to check if suppress_filters is passed or not.
+		$isUsed     = false;
+		$arrayValue = '';
+
+		// Retrieve the value parameter's details.
+		$argumentData = $this->phpcsFile->findNext( PHP_CodeSniffer_Tokens::$emptyTokens, $parameters[1]['start'], ( $parameters[1]['end'] + 1 ), true );
+
+		/**
```

Comment:
Here and in quite a few other places: `/**` => `/*`

This is not a docblock, but a multi-line inline comment ;-)